CHANGELOG.md

```diff
@@ -2,7 +2,11 @@
 
 ## Released
 
-### [0.4.0] - 2023-12-24
+### [0.5.0] - 2023-12-18
```

Comment:
If you're updating this, do you also need to update `lib/govuk-forms-markdown/version.rb`?

Reply:
Good catch, thank you. I've pushed up an update that addresses this.